src/GitHubUtil.js

```diff
@@ -101,19 +109,20 @@ async function getReviews(prNum) {
     let params = commonParams();
     params.pull_number = prNum;
 
-    const reviews = await GitHub.paginate(GitHub.rest.pulls.listReviews, params);
-    logApiResult(getReviews.name, params, {reviews: reviews.length});
-    // TODO: implement rateLimitedPromise() for 'paginated' results
-    return reviews;
+    const pages = await GitHub.paginate(GitHub.rest.pulls.listReviews, params, (response) => response);
+    const result = combinePages(pages);
+    logApiResult(getReviews.name, params, {reviews: result.data.length});
+    return await rateLimitedPromise(result);
```

Comment:
This does not look like the right rate limiting place to me: I would expect us to pause after each page rather than at the end of all pages. Am I missing something?